src/pages/LoginRegister/ForgotPasswordPage.jsx

```diff
@@ -27,7 +45,10 @@ export default function ForgotPasswordPage() {
             className="LoginRegisterForm_button"
             id="sub_btn"
             type="submit"
-          >
+            onClick = { () => {
+              const email = document.getElementsByClassName('userEmail').value
```

Comment:
Avoid imperative code like this.

Either useRef to get the element, or store the the value in state.